Sherlock/sherlock.h

```diff
@@ -83,6 +90,22 @@ CURRENT_STRUCT(SherlockSchema) {
   CURRENT_DEFAULT_CONSTRUCTOR(SherlockSchema) {}
 };
 
+CURRENT_STRUCT(SubscribableSherlockSchema) {
+  CURRENT_FIELD(type_id, current::reflection::TypeID);
+  CURRENT_FIELD(namespace_name, std::string, constants::kDefaultNamespaceName);
```

Comment:
Are these default values necessary?

I'd rather leave them blank, to make sure an uninitialized `SubscribableSherlockSchema` can't be confused with an initialized one from its JSON view. For the same reason, I'd initialize `type_id` to some `current::reflection::TypeID::UninitializedType`.